src/main/kotlin/game/racingcar/view/OutputView.kt

```diff
@@ -1,16 +1,21 @@
 package game.racingcar.view
 
-import game.racingcar.dto.LocationSnapshot
+import game.racingcar.dto.RacingCarPlayResult
 
 object OutputView {
 
-    fun printOutputToConsole(snapshots: List<LocationSnapshot>) {
+    fun printOutputToConsole(racingCarPlayResult: RacingCarPlayResult) {
         println("실행 결과")
-        snapshots.forEach { locationOfCars ->
-            locationOfCars.locations.forEach { location ->
+        racingCarPlayResult.locationSnapshots.forEach { locationOfCars ->
+            locationOfCars.locations.forEach { snapshot ->
+                val (name, location) = snapshot
+                print("$name : ")
                 repeat(location) { print("-") }
```

Comment:
String.repeat() 라는 확장함수도 있답니다 :) 취향에 맞게 취사선택 하시면 되겠네요!
```suggestion
                print("$name : ")
                print("-".repeat(location))
```

Reply:
오 몰랐던 부분인데 새로 알게되었네요!!
적용해볼게요!